source/hic_hal/gpio.h

```diff
@@ -22,6 +22,7 @@
 #ifndef GPIO_H
 #define GPIO_H
 
+#include "stdbool.h"
```

Comment:
`stdbool.h` should be included using angle brackets, i.e. `#include <stdbool.h>`, since it's a system header.